libs/auth-api-lib/src/lib/services/delegations.service.ts

```diff
@@ -92,6 +97,65 @@ export class DelegationsService {
         },
     )
   }
+
+  async create(delegation: DelegationDTO): Promise<Delegation | null> {
+    this.logger.debug('Creating a new delegation')
+    const id = uuid()
+    return await this.delegationModel.create({ id: id, ...delegation })
+  }
```

Comment:
Should we validate the toNationalId further? In natreg?

Or filter that it's not a company since we do not have a systems to handle delegations where toNationalId is a company.

Reply:
It might be nice if the IDS included a claim indicating if the current user is a company. So we're not doing "kennitala" checks all over the place.

Reply:
https://github.com/island-is/identity-server.web/issues/274.